README.md

```diff
@@ -22,6 +22,35 @@ cls.fit(X_train, y_train)
 predictions = cls.predict(X_test)
 ```
 
+## auto-sklearn acceleration with sklearnex
+
+You can accelerate auto-sklearn with [sklearnex](https://github.com/intel/scikit-learn-intelex). It is extension that speedups sklearn algorithms through patching with highly optimized machine learning kernels.
+
+Install sklearnex with pip or conda:
```

Comment:
I think it makes sense to refer to detailed install instructions/supported configurations

Reply:
System reqs are referred now